### ISSUE TITLE
Drag and drop seems broken on IE11, Microsoft Edge

### ISSUE THREAD

**Issue by jimitndiaye** [elsix/angular-jointjs-graph-demo#5]
Failure seems to happen in angular-jointjs-graph.js, specifically in the `setData` function below:

```
    el.addEventListener('dragstart', function (e) {
      e.dataTransfer.effectAllowed = 'copy';
      e.dataTransfer.setData('entity-attribtutes', JSON.stringify(el.dataset));
```

May have something to do with http://stackoverflow.com/questions/11927309/html5-dnd-datatransfer-setdata-or-getdata-not-working-in-every-browser-except-fi

Haven't been able to pinpoint exactly why the dataTransfer block is being locked; trying to isolate it with simpler drag/drop scenarios hasn't been successful (it works in those scenarios, e.g. https://jsfiddle.net/crl/rguLw8z2/1/).

**Comment by npetkov** [comment]
Could you please provide a trace as well? In which browser/version are you experiencing the problem?

**Comment by npetkov** [comment]
Ah, my fault, I see now. I haven't tested the framework on Microsoft products at all. If I can get my hands on a Windows machine soon I'll try and investigate what's going on there (hint: I suspect IE/Edge has some custom behavior when working with HTML5 data attributes...).

**Comment by jimitndiaye** [comment]
I think I might have figured it out... the jsfiddle I linked to previously can be gotten to fail by using any media type other than the standard ones, which the angular-jointjs-graph.js clearly does in using `entity-attributes` rather than something like `text\plain`.

**Comment by npetkov** [comment]
You explanation seems reasonable. It's fairly probable that Chrome/Firefox/Safari etc. simply ignore this. I'll fix it and will release a new version in the next days. Thanks for your feedback!

**Comment by jimitndiaye** [comment]
Problem is that specific event handler tries to set two different data payloads: `entity-attributes` and `pointer-offset`
Here's the stack trace:
SCRIPT65535: Unexpected call to method or property access.
angular-jointjs-graph.js (63,11)

**Comment by npetkov** [comment]
Would you be willing to submit a PR that implements a compliant behavior on trident browsers? I have a really limited experience with IE/Edge...

**Comment by npetkov** [comment]
Ok, I have a fix in the works and I guess I'll push a new release today, so this one is just to give you some information on what exactly was broken in IE/Edge. To start off, Edge doesn't seem to accept multiple data transfer items per event, even though the documentation states otherwise (this could have been fixed in a later Windows build; I use `10586.0`). It seems at least that the item format may be arbitrary (which is what the standard defines, as far as I can tell - see the documentation for the `setData` method [here](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/setData). In Internet Explorer 10/11 however, the item format must be `text`. You'll quickly find questions related to this strange behavior around the web.
